Makefile

```diff
@@ -33,8 +33,9 @@ install-safety:
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
 	# 12 Dec 2023: 51668 is fixed with >= 2.0.0b1 of SQLAlchemy. Ongoing refactor to upgrade.
 	# 6 June 2024: 70612 vulnerability found with jinja2 version 3.1.3
+	# 14 Aug 2024: 71600 found in gunicorn version 22.0.0, will be addressed in regular dependencies update
```

Comment:
It doesn't really matter, but apparently Mack is from the future.

Reply:
Man - I was like 100% sure it was the 14th.

Reply:
Somewhere in the world, it is already the 14th